var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -276,7 +292,7 @@ def bootstrap_args(self):
 
         # When building our own private copy of curl then we need to properly
         # enable / disable oepnssl
-        if '+ownlibs' in spec:
```

Comment:
This change is confusing for me - it seems like we are reversing the current logic?

Reply:
Oh good catch, missed that in the refactor after I fixed Zlib.